package.json

```diff
@@ -74,5 +74,15 @@
     "hooks": {
       "pre-commit": "npm-run-all -ln --parallel lint test"
     }
+  },
+  "dependencies": {
+    "@material-ui/core": "^4.11.3",
+    "@material-ui/icons": "^4.11.2",
```

Comment:
Not too familiar with this, does it mean you're using a different widgets library that wasn't used previously?

Reply:
Yeah we needed some icons while loading , for tables sort icons and all 
Can i share you some screenshot how it looks